### DISCUSSION TITLE
大世界是不是得等它开荒完了才能把其他选项勾上

### DISCUSSION THREAD

**Discussion by Moment0901** [LmeSzinc/AzurLaneAutoScript#1585] [Q&A] [answered]
我把“每月开荒”和“大世界每日”什么的都勾上了，结果他先做了每日，然后因为地图还没压制而出错了

**Comment by LmeSzinc** [comment, marked as answer]
> 大世界是不是得等它开荒完了才能把其他选项勾上

不是

> 我把“每月开荒”和“大世界每日”什么的都勾上了，结果他先做了每日，然后因为地图还没压制而出错了

因为你的大世界主线没有做完，做完了之后全港口是开放的，或者是没有购买并使用5000油，但是在alas里勾上了已经使用了5000油

**Reply by Moment0901** [reply]
看了下应该是偶然的bug，现在正常运行了

**Reply by Moment0901** [reply]
还有就是，“ 每月开荒”这个任务还在运行的时候，他就已经变成“等待中”了，虽然好像不影响运行但是不知道是不是bug

**Reply by LmeSzinc** [reply]
UI显示的问题，不影响实际运行
